BREAKING.md

```diff
@@ -6,6 +6,12 @@ recommended not to use the 'latest' Docker image tag blindly but pick a version
 and read this documentation before upgrading. This is where you will get information about
 breaking changes and about what you should do to overcome those changes.
 
+## Breaking in v4.24.0
```

Comment:
Can we include this under a subheading:

`### Deprecation Notice(s)`

Just for consistency with previous notices.